### DISCUSSION TITLE
Funds were not transferred from the exchange to the Chia wallet

### DISCUSSION THREAD

**Discussion by Saxus89** [Chia-Network/chia-blockchain#14010] [Support]
Hello. 09/14/2022 withdrew 2.08 xch from the Okx exchange to the Chia wallet. The coins have not yet been received, the Okx platform says that the transaction has been completed, but there is no information about the transaction on the Chia wallet itself, the coin count is also 0. I took the transfer address from the one specified in the wallet. Is it possible to understand this situation? I apologize in advance for the online translation.

**Comment by Bodman1** [comment]
> Hello. 09/14/2022 withdrew 2.08 xch from the Okx exchange to the Chia wallet. The coins have not yet been received, the Okx platform says that the transaction has been completed, but there is no information about the transaction on the Chia wallet itself, the coin count is also 0. I took the transfer address from the one specified in the wallet. Is it possible to understand this situation? I apologize in advance for the online translation.

I have the same problem, how did you solve yours?
